tests/conftest.py

```diff
@@ -81,6 +81,87 @@ def fridge_ds(samples_source, fridge_class_map) -> Tuple[Dataset, Dataset]:
     return train_ds, valid_ds
 
 
+@pytest.fixture(scope="module")
+def pennfudan_ds(samples_source) -> Tuple[Dataset]:
```

Comment:
Let me already comment here before you continue, testing custom parsers specific to a dataset should only be done in `icedata`. Icevision should not have any idea about pennfudan.

On icevision we have parsers for common formats like VOC that are not tied to a dataset. Basically what I'm trying to say is that if you're having to create a custom parser on testing it should not be on icevision. This is also related to #571  where something similar is happening

Reply:
Understood. Shall I just create a couple of dummy records then and remove pennfudan?
I thought it would be easier to run tests on some real-life samples.

Reply:
@lgvaz :)

Reply:
sorry for the delay! completely missed it first time, hectic day this has been

so so, you can use the samples we have for coco or voc =)

Reply:
Thanks!

Reply:
@lgvaz how about the latest commit?
Is it more along the expected lines?

Reply:
Perfect!!!

Another option for masks, if you prefer to use real-life samples, is to use the segmentation masks we have for voc or coco. The same way we create `fridge_ds` we can create `voc_mask_ds` or something like that

Reply:
I think I like the hand-crafted toy `records`. They make debugging really easy and they are very intuitive.